external-plugins/rehearse/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14.6-alpine3.12 as builder
+
+RUN mkdir /build
+COPY plugin/ /build
+RUN cd /build/ && go build -v .
+
+FROM alpine:3.12.0
```

Comment:
We can even use scratch

Reply:
True, but will an admin ever need to login to debug? Unless there's a truly compelling reason to strip this down to nothing, it would be a good idea to leave at least some way to execute a shell inside the pod's environment.

Reply:
maybe we don't even need a container to run this, this is pure golang we can run it with `go run` and it will compile it too like scripting golang.

Reply:
@qinqon it's a web-server that is supposed to run on K8s. We need a container.
I can't run it in a scratch container because I need git to be installed and as Stu mentioned already,
we want the ability to SSH into that container to debug.